### PR TITLE
initial docs for scaling

### LINKED ISSUE #242: Documentation for scaling behaviors
We should write some docs on how scaling works in watchbot 4, after we determine what we're going to do about `maxSize` (refs #241)

cc/ @mapbox/platform-engine-room

### PR DESCRIPTION
**Pull request by brendanmcfarland**
Adds description of scaling behaviors on the Watchbot Service. Tried to keep it brief with examples.

comments + edits + questions are very welcome @mapbox/platform-engine-room 

looking to close #242

### CODE REVIEW

File: docs/scaling-in-watchbot.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+The Watchbot 4 service scales on the SQS metrics ApproximateNumberOfMessagesVisible and ApproximateNumberOfMessagesNotVisible.
+
+### Scale Up / Scale Out
+
+The Scale Up policy is active when the SQS ApproximateNumberOfMessagesVisible metric is greater than zero. Scale up occurs in 10% increments of maxSize with a five minute cooldown between scaling activity. With any maxSize, there is a maximum scale up of 100 tasks and a minimum scale up of 1 task. For example: A stack with a maxSize of 50 will scale up 5 tasks every five minutes while ApproximateNumberOfMessagesVisible is greater than zero. A stack with a maxSize of 4 will scale up 1 task per scaling action, and finally a stack with a maxSize of 2000 will scale up in steps of 100 tasks per action.
+
+### Scale Down / Scale In
+
+The Scale Down policy is active whenever the SQS ApproximateNumberOfMessagesNotVisible metric is greater than zero. When ApproximateNumberOfMessagesNotVisible is zero - the stack will scale down all tasks above the minSize value.
```
Comment (arunasank) [inline docs/scaling-in-watchbot.md]: Do we mean to say that the Scale down policy is inactive when the `ApproximateNumberOfMessagesNotVisible` metric > 0, and is triggered when the  `ApproximateNumberOfMessagesNotVisible` = 0? We should also maybe reword the last sentence to say the "stack will scale itself down to the `minsize`" or so to make it clearer.

File: docs/scaling-in-watchbot.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+### Scale Up from Min Size 0
```
Comment (arunasank) [inline docs/scaling-in-watchbot.md]: Thanks for adding this as an extra reference. 🙌

File: docs/scaling-in-watchbot.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+## Scaling
+
+The Watchbot 4 service scales on the SQS metrics ApproximateNumberOfMessagesVisible and ApproximateNumberOfMessagesNotVisible.
+
+### Scale Up / Scale Out
+
+The Scale Up policy is active when the SQS ApproximateNumberOfMessagesVisible metric is greater than zero. Scale up occurs in 10% increments of maxSize with a five minute cooldown between scaling activity. With any maxSize, there is a maximum scale up of 100 tasks and a minimum scale up of 1 task. For example: A stack with a maxSize of 50 will scale up 5 tasks every five minutes while ApproximateNumberOfMessagesVisible is greater than zero. A stack with a maxSize of 4 will scale up 1 task per scaling action, and finally a stack with a maxSize of 2000 will scale up in steps of 100 tasks per action.
```
Comment (arunasank) [inline docs/scaling-in-watchbot.md]: These examples are useful for understanding decimal/non decimal increments and scaling when the maxsize is very large. 👌

File: docs/scaling-in-watchbot.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+## Scaling
+
+The Watchbot 4 service scales on the SQS metrics ApproximateNumberOfMessagesVisible and ApproximateNumberOfMessagesNotVisible.
```
Comment (arunasank) [inline docs/scaling-in-watchbot.md]: nitpick: Can we format all the metric names, so they stand out more? : )

**Comment by brendanmcfarland** [comment]
@arunasank thanks for the review! Made some edits and I'm going to merge so we get more updated docs out there for dev's upgrading.
